src/classlibnative/float/floatdouble.cpp

```diff
@@ -123,6 +123,11 @@ FCIMPL1_V(double, COMDouble::Cbrt, double x)
     return (double)cbrt(x);
 FCIMPLEND
 
+#if defined(_MSC_VER) && defined(_TARGET_AMD64_)
+// The /fp:fast form of `ceil` for AMD64 does not correctly handle: `-1.0 < value <= -0.0`
+#pragma float_control(precise, on )
```

Comment:
Is the plan to remove these pragmas once the underlying CRT bugs are fixed?

Reply:
I have logged https://github.com/dotnet/coreclr/issues/19739 to track removing these workarounds after the corresponding internal bugs are closed